components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -295,7 +315,14 @@ - (BOOL)shouldPresentFullScreen {
  the default value becomes 1.0.
  */
 - (CGFloat)initialDrawerFactor {
-  return [self shouldPresentFullScreen] ? 1 : (CGFloat)0.5;
+  if ([self shouldPresentFullScreen]) {
+    return 1;
+  }
+  return _initialDrawerFactor;
+}
+
+- (void)setInitialDrawerFactor:(CGFloat)initialDrawerFactor {
```

Comment:
if the setter does nothing you can remove it and even remove the synthesize if i recall correctly

Reply:
Removed